macros/record_dbt_project_models.sql

```diff
@@ -65,7 +66,7 @@
           {% for result in batch_results %}
             (
               '{{ result.node.name }}',
-              {% if result.relation_name is defined %}'{{ result.relation_name}}'{% else %}null{% endif %},
+              '{{ result.node.unique_id }}',
```

Comment:
so this is a little hack-y but relation_name isn't being populated anyways and we need to include the unique_id in the join.